vmdb/db/migrate/20141126161823_convert_show_refresh_button_and_load_values_on_init_to_real_columns_for_dialog_fields.rb

```diff
@@ -0,0 +1,28 @@
+class ConvertShowRefreshButtonAndLoadValuesOnInitToRealColumnsForDialogFields < ActiveRecord::Migration
+  class DialogField < ActiveRecord::Base
+    serialize :options
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :dialog_fields, :show_refresh_button, :boolean
+    add_column :dialog_fields, :load_values_on_init, :boolean
+
+    DialogField.all.each do |dialog_field|
+      dialog_field.show_refresh_button = dialog_field.options[:show_refresh_button]
+      dialog_field.load_values_on_init = dialog_field.options[:load_values_on_init]
```

Comment:
The options hash keys should be removed once the data is migrated to the dedicated columns.
```ruby
dialog_field.show_refresh_button = dialog_field.options.delete(:show_refresh_button)
dialog_field.load_values_on_init = dialog_field.options.delete(:load_values_on_init)
```